src/Compatibility/Microsoft.DotNet.ApiCompatibility/Abstractions/Filtering/AccessibilityFilter.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.DotNet.ApiCompatibility.Abstractions
+{
+    internal class AccessibilityFilter : ISymbolFilter
+    {
+        private readonly bool _includeInternalSymbols;
+
+        internal AccessibilityFilter(bool includeInternalSymbols)
+        {
+            _includeInternalSymbols = includeInternalSymbols;
+        }
+
+        public bool Include(ISymbol symbol) =>
```

Comment:
Take note of this scenario: https://github.com/dotnet/arcade/blob/3dd26504b456b68c2e3edd18ebf159d0b5486135/src/Microsoft.Cci.Extensions/Filters/PublicOnlyCciFilter.cs#L55-L66

And this one: https://github.com/dotnet/arcade/blob/3dd26504b456b68c2e3edd18ebf159d0b5486135/src/Microsoft.Cci.Extensions/Filters/PublicOnlyCciFilter.cs#L82-L93

Maybe those are other filters but they did matter for our existing API compat and GenAPI analysis.